app/messaging/create-message.js

```diff
@@ -0,0 +1,8 @@
+export const createMessage = (body, type, options) => {
```

Comment:
Realise you've just copied this off the other places that do this, but personally I hate this pattern of having an entire function, in an entirely separate file that just adds a single property into an object. Can we maybe simplify and remove this, and just add the source property in the place we create this originally?